package/kedro_viz/models/experiment_tracking.py

```diff
@@ -51,13 +51,12 @@ class TrackingDatasetGroup(str, Enum):
     JSON = "json"
 
 
-# pylint: disable=line-too-long
 TRACKING_DATASET_GROUPS = {
```

Comment:
```suggestion
# The keys will match a dataset type with any prefix, e.g. 
# plotly.plotly_dataset.PlotlyDataSet would include the following:
# kedro.extras.datasets.plotly.plotly_dataset.PlotlyDataSet
# kedro_datasets.plotly.plotly_dataset.PlotlyDataSet
# my.custom.path.to.plotly.plotly_dataset.PlotlyDataSet
TRACKING_DATASET_GROUPS = {
```

Just for my future reference so I don't forget what we've done here really...

(please check the comment is correct based on my understanding of the code!)